enhancements/baremetal/continuous-hw-introspection.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Continuous HW introspecton of Baremetal IPI hosts
```

Comment:
Nit: could we expand HW to `hardware` everywhere in this request? All reviewers may not immediately understand

Reply:
Done.